components/adv-post-form.js

```diff
@@ -311,16 +311,6 @@ export default function AdvPostForm ({ children, item, sub, storageKeyPrefix })
             }
               name='crosspost'
             />}
-          {me && itemType === 'poll' &&
-            <Checkbox
-              label={
-                <div className='d-flex align-items-center'>randomize order of poll choices
-                </div>
-            }
-              name='randPollOptions'
-              checked={formik?.values.randPollOptions}
-              onChange={formik.handleChange}
-            />}
```

Comment:
`AdvPostForm` is used by every `Item` form. We have an "escape hatch" for members to be added to the `AdvPostForm` accordian via the `children` prop.

Reply:
Also the formik stuff was unecessary afaict